### PR TITLE
Update config_rb_server_optional_settings.rst

### PR DESCRIPTION
**Pull request by trickyearlobe**
### Description

Documents optional ```data_collector['health_check']``` setting added to ```chef-server.rb``` by PR1704.

### Issues/PR's Resolved

Issue: https://github.com/chef/chef-server/issues/1703
PR: https://github.com/chef/chef-server/pull/1704
Related PR: https://github.com/chef/chef-server/pull/1716

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

Signed-off-by: Richard Nixon <rnixon@chef.io>

### CODE REVIEW

**Comment by irvingpop** [comment]
This should be merged after referenced PR is merged _and_ a new chef-server release is promoted to stable

**Review by irvingpop — APPROVED** [review]

**Comment by mjingle** [comment]
@trickyearlobe Hi! I noticed there was some movement on https://github.com/chef/chef-server/pull/1704. Please let us know if the documentation still matches what was implemented. Thank you in advance!

**Comment by tas50** [comment]
The PR was merged, but the release hasn't been made yet.

**Comment by trickyearlobe** [comment]
But to answer @mjingle question, yes, the doc updates still match the PR content that was merged, but not yet released.

**Comment by kagarmoe** [comment]
Hi @trickyearlobe ! Was this code merged?

**Comment by trickyearlobe** [comment]
@kagarmoe , yes, its shipped in 13.0.47

File: chef_master/source/config_rb_server_optional_settings.rst [hunk 1]
```diff
@@ -224,6 +224,9 @@ This configuration file has the following settings for ``data_collector``:
    The maximum connection duration (in seconds) for the HTTP connection pool that is used by the data collector. Default value: `"{70, sec}"`.
 ``data_collector['ibrowse_options']``
    An array of comma-separated key-value pairs of ibrowse options for the HTTP connection pool that is used by the data collector. Default value: `"[{connect_timeout, 10000}]"`.
+``data_collector['health_check']``
+   A boolean that controls whether the data collector health is included in the overall health at the ``_status`` endpoint. This setting when set to ``false`` is useful to prevent perfectly healthy front end Chef cluster members being reported as failed and removed from a load balancer pool when the ``data_collector['root_url']`` cannot be reached. Default value: ``true``.
```
Comment (IanMadd) [inline chef_master/source/config_rb_server_optional_settings.rst]:
```suggestion
   A boolean that controls whether the data collector health is included in the overall health at the ``_status`` endpoint. When set to ``true``, Chef Infra Server will report that healthy front end Chef HA cluster members have failed when the `data_collector['root_url']`` cannot be reached. As a result, the load balancer will remove those members from the load balancer pool. Default value: ``true``.
```

**Review by IanMadd — CHANGES_REQUESTED** [review]
I changed ``false`` to ``true`` and updated the other text to make it a bit more clear. Let me know what you think @trickyearlobe.

File: chef_master/source/config_rb_server_optional_settings.rst (hunk 1 above)
Comment (trickyearlobe) [reply chef_master/source/config_rb_server_optional_settings.rst]: Looks great, ship it ;-)

**Review by IanMadd — APPROVED** [review]

**Pull request merged**